dmoj/settings.py

```diff
@@ -161,6 +161,9 @@
 FONTAWESOME_CSS = '//maxcdn.bootstrapcdn.com/font-awesome/4.3.0/css/font-awesome.min.css'
 DMOJ_CANONICAL = ''
 
+DMOJ_DISABLE_COMMENTS = False
```

Comment:
I think it makes the code a little clearer (and removes the double-negatives) if we had these be `DMOJ_ENABLE_{COMMENTS,SOCIAL} = True`.

Reply:
Done :)